lib/driver.js

```diff
@@ -120,7 +120,7 @@ class EspressoDriver extends BaseDriver {
         autoLaunch: true,
         adbPort: DEFAULT_ADB_PORT,
         androidInstallTimeout: 90000,
-        systemPort: 8080,
+        systemPort: 6791,
```

Comment:
shouldn't it be set to DEVICE_PORT?

Reply:
@mykola-mokhnach yeah, my bad :) 

On second thought, do we really need to give _systemPort_ in defaultOpts ?

https://github.com/appium/appium-espresso-driver/blob/master/lib/driver.js#L144 this anyways finds free port and sets.

Reply:
FYI, UIA2 does not _systemPort_ in defaults

Reply:
I think we should keep the default. It should be fine.

Reply:
We shouldn't have a default for `systemPort`. We should, by default, do what @saikrishna321 talked about and have it find a free port.

The port that's used on the actual device is a hardcoded port, 6791, which shouldn't cause any issues so as long as the Android device isn't also using port 6791 (which is extremely unlikely).

Reply:
This is the block of code:

```this.opts.systemPort = this.opts.systemPort || await findAPortNotInUse(SYSTEM_PORT_RANGE[0], SYSTEM_PORT_RANGE[1]);```

If we set a default, it will never call `findAPortNotInUse`. So yes, we should remove the default.